test/FunctionalTests/Microsoft.OData.Edm.Tests/Csdl/EdmAnnotationsTargetTests.cs

```diff
@@ -0,0 +1,862 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="EdmAnnotationsTargetTests.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+#if NETCOREAPP3_1
+using System.Text.Encodings.Web;
+using System.Text.Json;
+#endif
+using System.Xml;
+using System.Xml.Linq;
+using Microsoft.OData.Edm.Csdl;
+using Microsoft.OData.Edm.Validation;
+using Microsoft.OData.Edm.Vocabularies;
+using Microsoft.OData.Edm.Vocabularies.V1;
+using Xunit;
+
+namespace Microsoft.OData.Edm.Tests.Csdl
+{
+    public class EdmAnnotationsTargetTests

```

Comment:
>EdmAnnotationsTargetTests [](http://example.com/codeflow?start=17&length=25)

it's better to only contains basic UT for EdmAnnotationsTarget in EdmAnnotationsTargetTests test class. Consider to move other writing/reading test cases into a new test class. You can make the CsdlReaderTest/CsdlWriterTest as partial class and create two new files only for EdmAnnotationsTarget reading and writing using the partial class of CsdlReaderTest/CsdlWriterTest. In this case, you don't need duplicate the WriteAndVerifyXml and WriteAndVerifyJson helper methods.